src/handlers/gestures/GestureDetector/Wrap.tsx

```diff
@@ -15,12 +18,22 @@ export class Wrap extends React.Component<{
       // correct viewTag to attach to.
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       const child: any = React.Children.only(this.props.children);
-      return React.cloneElement(
+
+      const clone = React.cloneElement(
         child,
         { collapsable: false },
         // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
         child.props.children
       );
+
+      if (Platform.OS === 'web') {
+        return (
+          <div ref={this.props.forwardedRef} style={{ display: 'contents' }}>
+            {clone}
+          </div>
+        );
+      }
+      return clone;
```

Comment:
What if we made an alternative Wrap component for web that was using `forwardRef` and pass the ref to the `div`. This way the second ref in the state for web isn't needed and I think the logic in other files remains unchanged.

Reply:
Let me know what do you think of current approach 😅